### ISSUE TITLE
??

### ISSUE THREAD

**Issue by EthnTuttle** [bitkarrot/UTXOracle.py#1]
Would this be a good thing to review to understand analyzing on-chain linkage? looking around for an open source on-chain surveillance tool to run against my own UTXO set.

**Comment by bitkarrot** [comment]
Yes. Its possible. This script is a copy from a post someone made on twitter.
